docs/introduction/get-started/p2p-private-offchain-txs.md

```diff
@@ -0,0 +1,97 @@
+In this section, we show you how to make off-chain transactions and send funds to another account using the Miden client. 
+
+!!! important "Prerequisite steps"
+    - You should have already followed all previous sections.
+    - You should have *not* reset the state of your local client. 
+
+## Create a second account
+
+!!! tip
```

Comment:
I wonder if we need a tutorial in both docs, here and in the client. 

To me, we could also keep it here as it should be prominent. What do you think?

Reply:
yeah, i think it's ok for now